backend/wait_for_db.py

```
@@ -1,20 +1,22 @@
 import time
 import MySQLdb
+import os
 from django.db import connections
```

Ruff (0.8.2)

`django.db.connections` imported but unused

Remove unused import: `django.db.connections`

(F401)